tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -37,27 +37,13 @@ services:
     privileged: true
     tty: true
     ports:
-      - "6899:6899"
       - "{{ awx_sdb_port_start }}-{{ awx_sdb_port_end }}:{{ awx_sdb_port_start }}-{{ awx_sdb_port_end }}"  # sdb-listen
 {% if cluster_node_count|int == 1 %}
+      - "6899:6899"
```

Comment:
not sure what this port does

Reply:
This is the port used to advertise connections to `sdb-listen` w/ UDP:

https://github.com/ryanpetrello/sdb/blob/master/sdb.py#L150-L154